src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        ColorSupplier colorSupplier = new ColorSupplier();
+        int numberOfBall = new Random().nextInt(100);
```

Comment:
You are creating a new instance of the `Random` class each time the `getRandomBall()` method is called. This is not efficient. Instead, you should create only one instance of `Random` and use it every time you need to generate a random number. You can do this by declaring a class-level `Random` variable.